templates/debug.hbs

```diff
@@ -1,19 +1,45 @@
 <style>
-    svg:first-child {
-        display: none;
+    .u-hidden{
+        display:none;
     }
-    svg {
+
+    code{
+        font-familly:monospace;
+        font-size:11px;
+        color:#666;
+        display:block;
+    }
+
+    .wrap{
+        display: flex;
+        flex-wrap: wrap;
+    }
+
+    div.item{
+        position:relative;
+        display: inline-block;
+        margin:10px;
+        width: 120px;
+        height: 100px;
+        text-align:center;
+    }
+    div.item>svg {
```

Comment:
This could be solved via a class, not a child selector.

Reply:
Also `.item` is probably enough, no need to overqualify with `div.item`.

Reply:
oki doki